Some systems require ncursesw.h to handle the Unicode characters that
csol uses, but we only have /usr/include/ncurses.h.  The CMake
CURSES_NEED_WIDE explicitly requires ncursesw.h, so we disable it.
--- ./CMakeLists.txt.orig 2021-05-08 13:30:58 +02:00
+++ ./CMakeLists.txt      2022-02-10 20:58:42 +01:00
@@ -9,7 +9,6 @@
 file(COPY games DESTINATION .)
 file(COPY themes DESTINATION .)
 
-set(CURSES_NEED_WIDE TRUE)
 find_package(Curses REQUIRED)
 include_directories("$(CURSES_INCLUDE_DIR)")
 
